frontend/src/components/ChaosAppBar/index.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { AppBar, Button, Toolbar } from "@mui/material";
+import HomeIcon from "@mui/icons-material/Home";
+import { CampaignName, LogoButton } from "./chaosAppBar.styled";
+
+const ChaosAppBar = (props) => {
```

Comment:
Not a huge fan of the name as the component, especially if/when we rename it to something more polished.